web-view/viewer/src/jsMain/kotlin/com/firsttimeinforever/intellij/pdf/viewer/application/Application.kt

```diff
@@ -220,27 +220,6 @@ class Application(private val viewer: ViewerAdapter) {
   }
 
   private fun rebindStandardEvents() {
-    viewer.viewerApp.asDynamic().unbindWindowEvents()
-    val boundEvents = viewer.viewerApp.asDynamic()._boundEvents
```

Comment:
_boundEvents is removed, see commit log 062acd1ef93892b326cdcb503e8b94bd40810d1f and https://github.com/mozilla/pdf.js/commit/702ee7b1e1775b6bfd57856beea77f650d6fa38a

Reply:
I see, I'm not quite sure why this code was here but everything seems to work fine still